app/tutorial_servo_control_can/src/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include "mbed.h"
+#include "CANMsg.h"
+#include "servo_control.h"
+#include "servo_control.cpp"
+
+AnalogIn potVoltageIN(PA_0);
+PwmOut servoPwmOut(PA_1);
+
+CAN can(CAN_RX, CAN_TX, ROVER_CANBUS_FREQUENCY);
+CANMsg txMsg;
+
+const float ANGLERANGE = 270;
+const float PULSEWIDTHRANGEMIN = 1;
+const float PULSEWIDTHRANGEMAX = 2;
+
+void initCAN()
+{
+    can.filter(RX_ID, ROVER_CANID_FILTER_MASK, CANStandard);
+}
+
+int main() {
+    
+    initCAN();
+
+    // initialize servo parameters
+    TutorialServo();

```

Comment:
You should be setting the pin with the constructor, you could also set the ranges in it as well